include/swift/AST/TBDGenRequests.h

```diff
@@ -70,7 +70,7 @@ class TBDGenDescriptor final {
   const TBDGenOptions &getOptions() const { return Opts; }
   TBDGenOptions &getOptions() { return Opts; }
 
-  const llvm::DataLayout &getDataLayout() const;
+  const llvm::DataLayout getDataLayout() const;
```

Comment:
The `TBDGenVisitor` holds a reference to the data layout, which it gets from here.
This returns a temporary now, so that reference is invalid.